r/NAMESPACE

```diff
@@ -31,6 +31,7 @@ S3method(is.na,Expression)
 S3method(is.na,array_expression)
 S3method(length,Array)
 S3method(length,ChunkedArray)
+S3method(length,Schema)
```

Comment:
Maybe this is just R convention, but it looks odd to me that Schema and Array share the `length` property. The two have the same name but are logically orthogonal (in the context of Schema it refers to column count but in an Array it refers to row count). It might be less confusing to just type out `length(names(Schema))` but I won't block on this

Reply:
Schema is logically a list of fields, and lists have length, so this makes sense to me. Array or other objects and whether they have length methods doesn’t seem relevant.